slack_sdk/oauth/installation_store/sqlite3/__init__.py

```diff
@@ -66,8 +69,11 @@ def create_tables(self):
                 user_token text,
                 user_scopes text,
                 incoming_webhook_url text,
+                incoming_webhook_channel text,
                 incoming_webhook_channel_id text,
                 incoming_webhook_configuration_url text,
+                is_enterprise_install text,
```

Comment:
I haven't checked the behavior yet but doing this way can work. Having three patterns - null / 1 (true) / 0 (false) - can be a pitfall. I would like to make this column not-null.
```suggestion
                is_enterprise_install boolean not null default 0,
```